src/app/basket/transaction-list/transaction-list.component.html

```diff
@@ -38,22 +38,20 @@
                     [attr.aria-labelledby]="'accordion-header-' + index"
                 >
                     <h4>Updated:</h4>
-                    <p>{{ item.updated }}</p>
-                    <h4>Statue-phase:</h4>
-                    <p>{{ item.status.phase }}</p>
-                    <h4>Statue-state:</h4>
+                    <p>{{ item.meta.updated }}</p>
+                    <h4>Status:</h4>
                     <p>{{ item.status.state }}</p>
                     <h4>Changes :</h4>
```

Comment:
@teone I'm seeing some alignment problems here - I'm wondering if the comma is necessary - also I think it would be better it the values could be right aligned - might require a table

![image](https://user-images.githubusercontent.com/10304208/160118084-19459e96-cfc4-439a-b66c-ba1d421cb134.png)

Reply:
Yeah, agreed it was pretty ugly, I replaced it with a table.